libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -32,14 +32,14 @@
             <strong>Published:</strong> {{ book.publishedDate | date: 'MM/dd/yyyy' }}
           </div>
           <p [innerHTML]="book.description"></p>
-          <div>
+          <div data-testing="add-to-list">
             <button
               mat-flat-button
               color="primary"
               (click)="addBookToReadingList(book)"
               [disabled]="book.isAdded"
             >
-              Want to Read
+            {{book.finished ? 'finished' : 'want to read'}}
```

Comment:
I would capitalize these words, as they already were ("Want to Read" instead of "want to read")

Reply:
Updated.